common/Dockerfile.tests

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6.1-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+COPY common/requirements.txt .
+
+RUN pip install -r requirements.txt
```

Comment:
Correct. Docker containers already isolate their environments so there's not really any point.